timeline/templates/timeline/side_timeline.html

```diff
@@ -45,7 +58,11 @@ <h2><i class="iconify" data-icon="mdi:clock-time-eight"></i> {{ timeline_title }
           </thead>
           <tbody>
             {% for event in object_list %}
-              {% include 'timeline/_list_item.html' %}
+              {% if timeline_mode == 'admin' %}
```

Comment:
We should simply check `timeline_mode` in the included template instead. Since the variable comes from the view context, `_list_item.html` should be able to access it directly without any extra steps needed.